.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
too late, if you add a file via `git add .` , then putting `launch.json` to .gitignore won't fix it as it's already tracked. You need to remove this file to make `.gitignore` work again.